static/src/javascripts/projects/common/modules/commercial/creatives/fabric-video.js

```diff
@@ -1,26 +1,34 @@
 define([
     'qwery',
+    'bonzo',
     'common/utils/fastdom-promise',
     'common/utils/detect',
     'common/utils/template',
+    'common/modules/commercial/creatives/add-tracking-pixel',
     'text!common/views/commercial/creatives/fabric-video.html'
-], function (qwery, fastdom, detect, template, fabricVideoStr) {
+], function (qwery, bonzo, fastdom, detect, template, addTrackingPixel, fabricVideoStr) {
     var fabricVideoTpl;
 
     return FabricVideo;
 
     function FabricVideo(adSlot, params) {
         var isUpdating = false;
+        var isMobile = detect.isBreakpoint({ max: 'phablet' });
```

Comment:
Yes that is counter-intuitive. It really is to detect the size of the viewport here. Thanks for spotting it!